docs/Installing.asciidoc

```diff
@@ -138,18 +138,31 @@ zypper dup --from devel_openQA_Leap --allow-vendor-change
 
 
 === Installation
-You can install the packages using these commands.
+You can install the main openQA server package using these commands.
 
 [source,sh]
 -------------------------------------------------------------------------------
-# openSUSE Leap 42.3+
+# openSUSE
 zypper in openQA
 
-
-# Fedora 23+
+# Fedora
 dnf install openqa openqa-httpd
 -------------------------------------------------------------------------------
 
+To install the openQA worker package use the following.
+
+[source,sh]
+-------------------------------------------------------------------------------
+# openSUSE
+zypper in openQA-worker
+-------------------------------------------------------------------------------
+
+Different convenience packages exist for convenience in openSUSE, for example:
+`openQA-local-db` to install the server including the setup of a local
+postgreSQL database or `openQA-single-instance` which sets up a webUI server,
```

Comment:
* `s/postgreSQL/PostgreSQL/`
* `s/webUI/web UI/` (Much more places use that spelling and it is consistent with web proxy in the next line.)

Reply:
done